project.godot

```diff
@@ -193,6 +193,11 @@ g_toggle_engulf={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":71,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
  ]
 }
+g_eject_engulfed={
```

Comment:
The new key needs to be added to `input_options.json` otherwise it is not reboundable and doesn't show up in the game settings.